src/Type/Php/FilterVarDynamicReturnTypeExtension.php

```diff
@@ -222,4 +224,27 @@ private function getFlagsValue(Type $exprType): Type
 		return $exprType->getOffsetValueType($this->flagsString);
 	}
 
+	private function canStringBeSanitized(Type $filterType, int $filterValue, ?Node\Arg $flagsArg, Scope $scope): bool
+	{
+		if (!$filterType instanceof StringType) {
+			return true;
+		}
+
+		// All validation filters match 0x100
+		// If it is a validation filter, the string will not be changed
+		if (($filterValue & 0x100) !== 0) {
```

Comment:
For readability maybe this magic constant should be extracted and documented at the definition level

Reply:
Done; let me know if that's sufficient.